lib/vanity/safe_yaml.rb

```diff
@@ -0,0 +1,17 @@
+require "yaml"
+
+module Vanity
+  module SafeYAML
+    begin
+      YAML.safe_load("---", permitted_classes: [])
+    rescue ArgumentError
+      def self.load(payload)
```

Comment:
Rather than defining different methods, I'd prefer to see a Boolean value  assigned to a constant like `SUPPORTS_PERMITTED_CLASSES`, and then that used within a singularly defined `.load` method.

Reply:
Thanks for the feedback. Will attempt to make this change.